src/Behat/Cores/Drupal8.php

```diff
@@ -306,6 +306,17 @@ public function getFileDestination($filename, $directory) {
   public function fileDelete($fid) {
     \Drupal::entityTypeManager()->getStorage('file')->load($fid)->delete();
   }
+  
+  /**
+   * {@inheritdoc}
+   */
+  public function getEntityByProperties($entity_type, $properties) {
```

Comment:
There already exists a method for this: https://github.com/Metadrop/behat-contexts/blob/master/src/Behat/Cores/Drupal8.php#L129 . This method must be removed, as the other method does the same and it has some more functionality.

Reply:
I'll delete this after merging.